robot.py

```diff
@@ -50,6 +50,17 @@ class Robot(magicbot.MagicRobot):
 
     offset_rotation_rate = 20
 
+    field_angles = {
+        "cargo front": 0,
+        "cargo right": math.pi / 2,
+        "cargo left": -math.pi / 2,
+        "loading station": math.pi,
+        "rocket left front": 0.52,  # measured field angle
```

Comment:
This seems quite close to 15°. Is this meant to be 15°?

Reply:
That is the (rounded) angle we measured on the field elements

Reply:
Blame FIRST for having the rocket at ~61 degrees